spec/lib/twingly/url/normalization_spec.rb

```diff
@@ -32,6 +34,25 @@
       expect(normalizer.normalize(url)).to eq([url])
     end
 
+    it "handles URL with punycoded SLD" do
+      url = "http://www.xn--4cab6c.se/"
+
+      expect(normalizer.normalize(url)).to eq([url])
+    end
+
+    it "handles URL with punycoded TLD" do
+      url = "http://www.example.xn--p1ai/"
+
+      expect(normalizer.normalize(url)).to eq([url])
+    end
```

Comment:
We should add test without subdomain here, I suspect it will fail... because of https://github.com/sporkmonger/addressable/issues/199